client/src/components/dashboard.js

```diff
@@ -1,12 +0,0 @@
-/**
- * This file exports a dashboard component, the page our users first land on upon signing in
- */
-import React from 'react';
-import {useSelector} from 'react-redux';
-
-export default function Dashboard() {
-    const userName = useSelector(state => {console.log(state); return state.user?.givenName});
-    return (
-        <div>Hi {userName}!</div>
-    )
-}
```

Comment:
Is this dashboard.js team Logiciel's commit? Are we deleting it, if we are you can ignore this comment.

Reply:
Erm I dont think it's our commit, but i guess we took over the dashboard task after other team created a welcoming-user-dashboardpage?

Reply:
Sure then if it's not used we can delete it